manifests/params.pp

```diff
@@ -55,40 +51,53 @@
   $whisper_workers                    = 1
   # Limits the number of whisper update_many() calls per second. 0 - no limit
   $whisper_max_updates_per_second     = 0
+  $whisper_max_creates_per_second     = 500
   $whisper_enabled                    = true
 
   # Limit of in-memory stored points (not metrics)
   $cache_max_size                     = 1000000
   # Capacity of queue between receivers and cache
-  $cache_input_buffer                 = 51200
+  # Strategy to persist metrics. Values: "max","sorted","noop"
+  #   "max" - write metrics with most unwritten datapoints first
+  #   "sorted" - sort by timestamp of first unwritten datapoint.
+  #   "noop" - pick metrics to write in unspecified order,
+  #            requires least CPU and improves cache responsiveness
+  $cache_write_strategy               = 'max'
 
-  $udp_listen                         = '0.0.0.0:2003'
```

Comment:
Why?

Reply:
Basing this change off the go-carbon.conf generated from the 0.13.0 deb install and current README.